app/views/users/show.html.haml

```diff
@@ -0,0 +1,27 @@
+=link_to 'Edit profile', edit_user_registration_path
+
+- attr = ['firstname', 'lastname', 'address', 'about', 'drivingage', 'age', 'phone', 'email']
+
+%h4 Name:
+%p= "#{current_user.firstname} #{current_user.lastname}"
+
+%h4 Email:
+%p= current_user.email
+
+%h4 Address:
+%p= current_user.address
```

Comment:
Using `current_user` here is not a good choice. What if a car owner wants to inspect someone's profile before they decide on renting their car out to him/her.

Reply:
done